lib/parsers/mp3_parser.rb

```diff
@@ -133,6 +136,10 @@ def call(raw_io)
 
   private
 
+  def tif?(io)
+    FormatParser::TIFFParser::HEADERS.include?(safe_read(io, 4))
```

Comment:
If we check against two literal bytebags, I would suggest copying them here instead of creating references between parsers - it is conceptually better, IMO, if the parsers stay independent from one another. Little duplication is likely a better choice here.

Reply:
@julik I understand that it would be good to stay independent but on the other side, it seems to me, that it's a responsibility of the `TIFFParser` to know what headers are related to it.

If I duplicate them and they change someday, the MP3Parser would be using the wrong information.

wdyt?

Reply:
Ok, got it!